.travis.yml

```diff
@@ -27,7 +27,4 @@ after_script:
 
 language: node_js
 node_js:
-  - "4"
-  - "5.6"
-  - "5.12"
```

Comment:
lets keep these around

Reply:
or at least 5.12

Reply:
@dustinsmith1024, 5.12 doesn't have `Proxy` which this implementation will need. I am hoping that we can update the major version and only support node-6 forward.